tests/testthat/test-helpers_jags.R

```diff
@@ -92,10 +92,15 @@ test_that("h_jags_get_data works as expected", {
   model <- h_get_logistic_log_normal()
 
   result <- h_jags_get_data(model, data, FALSE)
+  expected_prior <- c(
+    h_slots(data, c("nObs", "y", "x")),
+    model@modelspecs(TRUE) # nolintr
+  )
   expected <- c(
     h_slots(data, c("nObs", "y", "x")),
-    model@modelspecs() # nolintr
+    model@modelspecs(FALSE) # nolintr
```

Comment:
in all these tests to make it better readable I would always say `from_prior = TRUE` etc. so with the argument name